Library/Homebrew/test/spec_helper.rb

```diff
@@ -146,6 +146,11 @@
   config.include(Test::Helper::Formula)
   config.include(Test::Helper::MkTmpDir)
 
+  # Enable aggregate failures by default
+  config.define_derived_metadata do |metadata|
+    metadata[:aggregate_failures] = true unless metadata.key?(:aggregate_failures)
```

Comment:
```suggestion
    metadata[:aggregate_failures] = true unless metadata.key?(:no_aggregate_failures)
```

Thoughts on this to avoid confusing anyone who's familiar with the default `:aggregate_failures` functionality?

Reply:
Let's review if it causes confusion.